Source/WebCore/dom/Element.cpp

```diff
@@ -1066,6 +1072,8 @@ static std::optional<std::pair<RenderElement*, LayoutRect>> listBoxElementScroll
 
 void Element::scrollIntoView(std::optional<std::variant<bool, ScrollIntoViewOptions>>&& arg)
 {
+    document().contentVisibilityDocumentState().updateContentRelevancyStatusForScrollIfNeeded(*this);
```

Comment:
This is making Document create a ContentVisibilityDocumentState even when it has no skipped content?

Reply:
Good point, fixed.